packages/fxa-auth-server/test/local/senders/email.js

```diff
@@ -319,6 +322,28 @@ const TESTS = [
       { test: 'notInclude', expected: 'utm_source=email' },
     ]]
   ])],
+  ['subscriptionRenewalReminderEmail', new Map([
+    ['subject', { test: 'equal', expected: `${MESSAGE.productName} automatic renewal notice` }],
+    ['headers', new Map([
+      ['X-SES-MESSAGE-TAGS', { test: 'equal', expected: sesMessageTagsHeaderValue('subscriptionRenewalReminder') }],
+      ['X-Template-Name', { test: 'equal', expected: 'subscriptionRenewalReminder' }],
+      ['X-Template-Version', { test: 'equal', expected: TEMPLATE_VERSIONS.subscriptionRenewalReminder }],
+    ])],
+    ['html', [
+      { test: 'include', expected: configHref('subscriptionTermsUrl', 'subscription-renewal-reminder', 'subscription-terms') },
+      { test: 'include', expected: configHref('subscriptionSettingsUrl', 'subscription-renewal-reminder', 'update-billing', 'plan_id', 'product_id', 'uid', 'email') },
+      { test: 'include', expected: configHref('subscriptionSupportUrl', 'subscription-renewal-reminder', 'subscription-support') },
+      { test: 'include', expected: `Your current subscription is set to automatically renew in ${MESSAGE.reminderLength} days. At that time, Mozilla will renew your ${MESSAGE.planIntervalCount} ${MESSAGE.planInterval} subscription and a charge of ${MESSAGE_FORMATTED.invoiceTotal} will be applied to the payment method on your account.` },
+      { test: 'notInclude', expected: 'utm_source=email' },
+    ]],
+    ['text', [
+      { test: 'include', expected: `${MESSAGE.productName} automatic renewal notice` },
+      { test: 'include', expected: `Your current subscription is set to automatically renew in ${MESSAGE.reminderLength} days. At that time, Mozilla will renew your ${MESSAGE.planIntervalCount} ${MESSAGE.planInterval} subscription and a charge of ${MESSAGE_FORMATTED.invoiceTotal} will be applied to the payment method on your account.` },
+      { test: 'notInclude', expected: 'utm_source=email' },
+    ]]
+  ]),
+  {updateTemplateValues: x => (
+    {...x, subscription: {planId: MESSAGE.planId, productId: MESSAGE.productId, ...x.subscriptions[0]}})}],
```

Comment:
I don't think this optional arg is necessary.

Reply:
It isn't if I add the `subscription` object to `MESSAGE` at the top of the file. I can do that instead.

Reply:
Oh this is the first email that gets a `subscription` in the template values.  Sorry about that.

Reply:
> 
> Edit: Assuming you are meaning I don't need to pass `updateTemplateValues` at all...

I did.